apps/src/templates/progress/progressHelpers.js

```diff
@@ -41,6 +36,32 @@ export function lessonIsVisible(lesson, state, viewAs) {
   return !isHidden || viewAs === ViewType.Teacher;
 }
 
+/**
+ * Treat the lesson as locked if either
+ * (a) it is locked for this user (in the case of a student)
+ * (b) it is locked for all students in the section (in the case of a teacher)
+ * (c) non-verified teacher or signed out user
```

Comment:
Do we always display cases (b) and (c) in the same way?  Case (b) seems different to me than the other two cases in the sense that the level isn't really "locked" for the teacher (i.e. the teacher can still access it).

Reply:
Ya I agree with you that they are different. I think your question makes me want to dig into this a little more to see if I can make the complicated logic that exists here even better. This function was step one but I think there is more that I can do